geomesa/src/test/scala/geotrellis/geotools/GeoMesaSimpleFeatureType.scala

```diff
@@ -0,0 +1,38 @@
+package geotrellis.geotools
+
+import geotrellis.vector.Geometry
+import geotrellis.proj4.{CRS => GCRS, WebMercator}
+
+import com.vividsolutions.jts.{geom => jts}
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder
+import org.locationtech.geomesa.accumulo.index.Constants
+
+import scala.reflect._
+
+object GeoMesaSimpleFeatureType {
+
+  val whenField  = "when"
+  val whereField = "where"
+
+  def apply[G <: Geometry: ClassTag](featureName: String, crs: Option[GCRS] = Some(WebMercator), temporal: Boolean = false) = {
+    val sftb = (new SimpleFeatureTypeBuilder).minOccurs(1).maxOccurs(1).nillable(false)
+
+    sftb.setName(featureName)
+    crs.foreach { crs => sftb.setSRS(s"EPSG:${crs.epsgCode.get}") }
```

Comment:
Is there a fallback if the CRS is not EPSG code? Not sure how common that is with vectors but in rasters we've seen that happen.